docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #177.    def DAMP(T, m, split_index):  # Add param <code>T_subseq_isconstant</code>  
</pre><p>Change to <code>split_idx</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>